src/applications/check-in/api/local-mock-api/mocks/v2/check-in-data/get.js

```diff
@@ -78,6 +80,8 @@ const createAppointment = (
     startTime.setHours(startTime.getHours() - 1);
   } else if (eligibility === 'INELIGIBLE_TOO_EARLY') {
     startTime.setHours(startTime.getHours() + 1);
+  } else if (uuid === aboutToExpireUUID) {
+    startTime.setMinutes(startTime.getMinutes() - 14);
```

Comment:
For consistency, do we want to bring in date-fns sub and add functions here?

Reply:
yeah, I like it :-)